frontend/packages/dev-console/src/components/health-checks/HealthChecksProbe.tsx

```diff
@@ -59,22 +61,28 @@ const HealthCheckProbe: React.FC<HealthCheckProbeProps> = ({ probeType }) => {
     if (healthChecks?.[probeType]?.enabled) {
       return (
         <>
-          <span className="odc-heath-check-probe__successText">
-            <GreenCheckCircleIcon /> {`${getHealthChecksProbeConfig(probeType).formTitle} Added`}
-          </span>
-          <Tooltip content="Remove" position="right">
-            <Button
-              className="pf-m-plain--align-left"
-              variant={ButtonVariant.plain}
-              onClick={handleDeleteProbe}
-            >
-              <MinusCircleIcon />
-            </Button>
-          </Tooltip>
+          <Button variant={ButtonVariant.plain} onClick={showProbe}>
```

Comment:
Please fix the alignment of the success text

![Screenshot from 2020-09-17 14-29-49](https://user-images.githubusercontent.com/22490998/93450174-49f8b980-f8f3-11ea-9d0c-b5d809e4932d.png)

Reply:
fixed!